### PR TITLE
Get Organization / Event Test Coverage to 100%

### PR DESCRIPTION
**Pull request by ajaygandecha**
This small PR adds more Pytests to get the tests for `OrganizationService` and `EventService` to 100%.

This PR also removes the unused `OrganizationNotFoundException`, which was causing the exceptions to not hit 100% test coverage.

### CODE REVIEW

File: backend/test/services/organization/organization_test.py [hunk 1]
```diff
@@ -81,6 +82,17 @@ def test_create_organization_as_root(organization_svc_integration: OrganizationS
     assert created_organization.id is not None
 
 
+def test_create_organization_id_already_exists(
+    organization_svc_integration: OrganizationService,
+):
+    """Test that the root user is able to create new organizations when an extraneous ID is provided."""
+    created_organization = organization_svc_integration.create(
+        root, to_add_conflicting_id
+    )
```
Comment (ajaygandecha) [inline backend/test/services/organization/organization_test.py:91]: @KrisJordan when creating this test, I wanted to ask if we want to start standardizing all features to use a `Draft` model to specifically avoid issues with IDs when creating items, including for organizations, events, etc. Thoughts? For now, I added these tests to get to 100% coverage before such a refactor is done more widely.

Comment (KrisJordan) [reply backend/test/services/organization/organization_test.py:91]: I think moving in this direction makes sense, but agree it's a larger refactoring that is orthogonal

**Review by KrisJordan — APPROVED** [review]
LGTM

**Review by KrisJordan — APPROVED** [review]
LGTM

**Pull request merged**